mslib/msui/msui_mainwindow.py

```diff
@@ -735,6 +738,85 @@ def handle_export_local(self, extension, function, pickertype):
         else:
             self.mscolab.handle_export_msc(extension, function, pickertype)
 
+    def copy_into_new_flight_track(self):
+        if self.local_active:
+            name = self.active_flight_track.name
+            template_copy = copy.deepcopy(self.active_flight_track.all_waypoint_data())
+        else:
+            name = self.mscolab.active_operation_name
+            template_copy = copy.deepcopy(self.mscolab.waypoints_model.all_waypoint_data())
+
+        # Create a new flight track from the waypoints' template.
+        self.new_flight_track_counter += 1
+        waypoints_model = ft.WaypointsTableModel(
+            name=name + f" (copy {self.new_flight_track_counter:d})")
+        # Make a copy of the template. Otherwise, all new flight tracks would
+        # use the same data structure in memory.
+        waypoints_model.insertRows(0, rows=len(template_copy), waypoints=template_copy)
```

Comment:
here we could add a flag which writes then in the chat a message, e.g. "copied data". The chat system messages can be seen as a log what happens to the operation 

https://github.com/Open-MSS/MSS/blob/develop/mslib/msui/flighttrack.py#L439